### ISSUE TITLE
[BUG]: Connection error when using Power BI

### ISSUE THREAD

**Issue by soumya67** [microsoft/coe-starter-kit#544]
**Describe the bug**
The error is while setting up PowerBI Dashboard per [https://docs.microsoft.com/en-us/power-platform/guidance/coe/setup-powerbi](url)

**Component (please tell us which flow or app you are experiencing issues with):**
PowerBI Dashboard

**To Reproduce**
1. Go to PowerBI Windows App
2. Open step for Configure the Power BI dashboard by working directly with the Power BI (.pbit) file
3. Provide environment URL
4. See error per image attached

**Expected behavior**
There should be no error

**Screenshots**
See image attached
![image](https://user-images.githubusercontent.com/12087685/120039386-d014e600-bfd2-11eb-8dfc-71a729d6956d.png)

**Comment by manuelap-msft** [comment]
Hello,

this is a duplicate of #341 - can you read through that thread and try the recommended solution approaches, such as checking if any ports are blocked?

Thank you
Manuela

**Comment by manuelap-msft** [comment]
Hello,

there is a limitation / issue with how this currently works in Dataverse for Teams, that the Engineering team is looking at. In the meantime, the below steps should unblock you:

1. Go to flow.microsoft.com
2. Select your Dataverse for Teams environment
3. Select Solution
4. Select Common Data Service Default Solution
5. Select + Add > Environment Variables
6. Select the existing Environment Variables from the managed solution that you want to update
7. After adding those, selected Environment Variables in the new menu navigation and you should then be able to add a default and current value

Thank you,
Manuela

**Comment by JeneferM-MSFT** [comment]
closing out as no further action for toolkit
